CHANGELOG.md

```diff
@@ -2,6 +2,49 @@
 
 <!-- next version -->
 
+## 0.104.0
+
+### 🛑 Breaking changes 🛑
+
+- `collector`: The new collector release enables `component.UseLocalHostAsDefaultHost` by default. This mainly affects the `otlpreceiver`. But may has side effects on components like the `healthcheck extension`. The `operator` will change IPs not explicitly configured in the `otlpreceiver` configuration such as `:4317` or empty fields to `0.0.0.0` during the upgrade routine and when creating a CR.
```

Comment:
Is there any link for collector we can put here?

Reply:
done